File: BACKEND/EditorialMicroservice/src/main/java/com/example/editorialmicroservice/entity/Editorial.java
package com.example.editorialmicroservice.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * Esta clase representa al objeto editorial
 */
@Entity
public class Editorial {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombre;
    private String web_url;

    public Editorial() {
    }

    public Editorial(String nombre, String web_url) {
        this.nombre = nombre;
        this.web_url = web_url;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String name) {
        this.nombre = nombre;
    }

    public String getWeb_url() {
        return web_url;
    }

    public void setWeb_url(String web_url) {
        this.web_url = web_url;
    }
}

File: FRONTEND/proyecto-bookstore/src/app/models/editorial.ts
/**
 * Esta clase representa al objeto de Editoriald e la libreria
 */
export class Editorial {
    /**Atributos de la clase */
    id?: number;
    nombre: string;
    web_url: string;
    /**
     * Constructor de la clase
     * @param nombre 
     * @param web_url 
     */
    constructor(nombre: string, web_url: string){
        this.nombre = nombre;
        this.web_url = web_url;
    }

}
